### PR TITLE
Enable linting and fix lint issues for app, util and logger

### PR DESCRIPTION
**Pull request by Feiyang1**

### CODE REVIEW

File: packages/util/src/crypt.ts [hunk 1]
```diff
@@ -154,34 +154,34 @@ export const base64 = {
    *
    * @param input An array of bytes (numbers with
    *     value in [0, 255]) to encode.
-   * @param opt_webSafe Boolean indicating we should use the
+   * @param optwebSafe Boolean indicating we should use the
```
Comment (hsubox76) [inline packages/util/src/crypt.ts]: Did you mean to get rid of the `opt_` entirely? https://engdoc.corp.google.com/eng/doc/devguide/js/styleguide.md?cl=head#features-functions-default-parameters

If not, it should still be camelCase I think.

File: packages/util/src/sha1.ts [hunk 2]
```diff
@@ -103,19 +103,19 @@ export class Sha1 {
...
    * Internal compress helper function.
    * @param {!number[]|!Uint8Array|string} buf Block to compress.
-   * @param {number=} opt_offset Offset of the block in the buffer.
+   * @param {number=} optOffset Offset of the block in the buffer.
    * @private
    */
-  compress_(buf, opt_offset?) {
-    if (!opt_offset) {
-      opt_offset = 0;
+  compress_(buf, optOffset?): void {
```
Comment (hsubox76) [inline packages/util/src/sha1.ts]: Should just be `offset` if we are following Google style guide (see previous comment)

File: packages/util/src/sha1.ts [hunk 3]
```diff
@@ -190,53 +190,53 @@ export class Sha1 {
     this.chain_[4] = (this.chain_[4] + e) & 0xffffffff;
   }
 
-  update(bytes, opt_length?) {
+  update(bytes, optLength?): void {
```
Comment (hsubox76) [inline packages/util/src/sha1.ts]: Same as above, `length`?

File: packages/util/src/crypt.ts (hunk 1 above)
Comment (Feiyang1) [reply packages/util/src/crypt.ts]: Good catch. Updated to `webSafe`.

Comment (Feiyang1) [reply packages/util/src/crypt.ts]: I think they mean `option` instead of `optional` which the google styleguide references to, but it is redundant.

**Pull request merged**